.github/workflows/msys2.yml

```diff
@@ -0,0 +1,98 @@
+name: Build on MSYS2
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
```

Comment:
Can we add these builds as a new job under the existing Build workflow instead of putting them in a different workflow file? Conceptually what is being tested here seems to be a pretty good match for what is is there. I'm not sure if it makes sens to merge them as part of a single matrix for one job or keep it as two jobs with their own matrices but at least being part of the same workflow makes more sense to me.

Reply:
done